src/overlays/actors/ovl_En_Anubice/z_en_anubice.c

```diff
@@ -354,15 +361,15 @@ void EnAnubice_Update(Actor* thisx, GlobalContext* globalCtx) {
     Vec3f sp3C;
     EnAnubice* this = (EnAnubice*)thisx;
 
-    if (this->actionFunc != EnAnubice_SetupDie && this->actionFunc != EnAnubice_Die &&
-        this->actor.shape.yOffset == 0.0f) {
+    if ((this->actionFunc != EnAnubice_SetupDie) && (this->actionFunc != EnAnubice_Die) &&
+        (this->actor.shape.yOffset == 0.0f)) {
         EnAnubice_Hover(this, globalCtx);
-        for (i = 0; i < 5; i++) {
+        for (i = 0; i < ARRAY_COUNT(this->flameCircles); i++) {
             flameCircle = this->flameCircles[i];
 
-            if (flameCircle != NULL && fabsf(flameCircle->actor.world.pos.x - this->actor.world.pos.x) < 60.0f &&
-                fabsf(this->flameCircles[i]->actor.world.pos.z - this->actor.world.pos.z) < 60.0f &&
-                flameCircle->timer != 0) {
+            if ((flameCircle != NULL) && (fabsf(flameCircle->actor.world.pos.x - this->actor.world.pos.x) < 60.0f) &&
+                (fabsf(this->flameCircles[i]->actor.world.pos.z - this->actor.world.pos.z) < 60.0f) &&
+                (flameCircle->timer != 0)) {
```

Comment:
why add back unnecessary parens?

Reply:
everyone has their own preferences regarding this, and we dont enforce one or the other. this looks more readable to me personally, but understand not everyone shares that opinion 
until when and if we decide to standardize one over the other, theres not really much that can be said

Reply:
I believe when it comes to warnings, if logic such as `&&` or `||` is not enclosed in parens, it will create a warning, so this is just preventing warnings. At least that is what I know in regards to how things are in MM.

Reply:
this doesnt induce warnings at least in oot, its purely preference. but yea i guess mm may have different warnings enabled

Reply:
I prefer less parenthesis usage as they look really noisy in longer conditionals, but I think we should enforce explicit parenthesis everywhere for the sake of consistency.

Reply:
and I say we shouldn't enforce a bad pattern simply because it was more comfortable for mips2c to generate.

Reply:
I mean, no lol. I just said above that I prefer explicit parenthesis, and clearly elliptic prefers it as well cause he added them here. And plenty more have said they prefer it. Its not just because of mips2c.
That said there are just as many that dont like them, its a pretty split issue.